apps/notion/src/connectors/users.ts

```diff
@@ -0,0 +1,82 @@
+import { NotionError } from './commons/error';
+
+type Person = {
+  email: string;
+};
+
+export type NotionUser = {
+  id: string;
+  object: string;
+  name: string;
+  person: Person;
+  type: string;
+};
+
+export type ElbaUser = {
+  id: string;
+  displayName: string;
+  email: string;
+  additionalEmails: string[];
+};
+
+type GetUsersResponseData = { results: NotionUser[]; next_cursor: string | null };
+
+export const getUsers = async (token: string, pageSize: string, sourceBaseUrl: string, page: string | null, notionVersion: string) => {
+  let url = `${sourceBaseUrl}/v1/users?page_size=${pageSize}`;
+  if (page) {
+    url = `${sourceBaseUrl}/v1/users?page_size=${pageSize}&start_cursor=${page}`;
+  }
```

Comment:
We can get ride of `sourceBaseUrl` argument and use the `env` directly.
Regarding the `version` argument, if it's always the same value pass, perhaps we can remove it as well and put the value directly in the header. This will make `getUsers` implementation simpler.
```suggestion
  const url = new URL(`{env.NOTION_API_BASE_URL}/v1/users`);
  url.searchParams.append('page_size', pageSize)
  if (page) url.searchParams.append('start_cursor', page)
```